src/Wave.cpp

```diff
@@ -109,16 +109,16 @@ Wave::Effects Wave::computeEffects(double seconds, Vector3d const wave_amplitude
     plane_wave_amplitude.Z() = 0;
 
     // Compute the wave's time coefficients
-    double wave_coeff_time_x = sin(M_PI*2*seconds*wave_frequency_world.X() + phase_x);
-    double wave_coeff_time_y = sin(M_PI*2*seconds*wave_frequency_world.Y() + phase_y);
-    double wave_coeff_time_z = sin(M_PI*2*seconds*wave_frequency_world.Z() + phase_z);
+    double wave_coeff_time_x = sin(M_PI*2*seconds*wave_frequency_world.X() + m_phase_x);
+    double wave_coeff_time_y = sin(M_PI*2*seconds*wave_frequency_world.Y() + m_phase_y);
+    double wave_coeff_time_z = sin(M_PI*2*seconds*wave_frequency_world.Z() + m_phase_z);
```

Comment:
rename these for something like `wave_force_scale_x`, time coefficient is not a good name